pywps/response/__init__.py

```diff
@@ -57,9 +63,13 @@ def _update_status(self, status, message, status_percentage):
         self.status_percentage = status_percentage
         store_status(self.uuid, self.status, self.message, self.status_percentage)
 
+    @abstractmethod
+    def _construct_doc(self):
+        ...
```

Comment:
```suggestion
        pass
```

Reply:
These are both equally accepted idioms in python3:
https://docs.python.org/3/library/abc.html
https://www.geeksforgeeks.org/what-is-three-dots-or-ellipsis-in-python3/